io.fairyproject.modules/platform.mc/module.protocol/src/main/java/io/fairyproject/mc/protocol/event/MCPlayerProtocolPacketEvent.java

```diff
@@ -0,0 +1,52 @@
+package io.fairyproject.mc.protocol.event;
+
+import com.github.retrooper.packetevents.event.ProtocolPacketEvent;
+import com.github.retrooper.packetevents.manager.server.ServerVersion;
+import com.github.retrooper.packetevents.protocol.ConnectionState;
+import com.github.retrooper.packetevents.protocol.packettype.PacketTypeCommon;
+import io.fairyproject.event.Cancellable;
+import io.fairyproject.mc.MCPlayer;
+import io.fairyproject.mc.event.trait.MCPlayerEvent;
+import org.jetbrains.annotations.NotNull;
+
+public class MCPlayerProtocolPacketEvent implements MCPlayerEvent, Cancellable {
```

Comment:
Why keyword "get" is never used?

Reply:
What do you mean by keyword "get?"

Reply:
Methods should contain a verb, for a lot of your getters in this PR the method miss "get" keyword, example: pos() our player() are just position & player getters but this do not really follow Java convention tbh

Reply:
For some reason I was fall in love to the idea of everything "fluent" including getters and setters.
But I guess you are right, stop fluent getter.